CanvasColorSpaceProposal.md

```diff
@@ -138,21 +138,35 @@ ImageBitmap objects are augmented to have an internal color space attribute of t
 
 IDL
 <pre>
+
+enum ImageDataStorageFormat {
```

Comment:
"type" instead of "format". The word format implies more than just type. e.g. the number of channels and/or their order.